static/css/s2.css

```diff
@@ -12993,6 +13138,39 @@ span.ref-link-color-3 {color: blue}
     font-size: 14px;
     line-height: 18px;
     color: #666666;
+}
+
+@media (max-width: 768px) {
+  .productsInner {
```

Comment:
Flexbox should be taking care of this for you?

Reply:
It wasn't working well until I added a media query.